pkg/scheduler/framework/plugins/nodeports/node_ports.go

```diff
@@ -76,6 +78,9 @@ func getContainerPorts(pods ...*v1.Pod) []*v1.ContainerPort {
 // PreFilter invoked at the prefilter extension point.
 func (pl *NodePorts) PreFilter(ctx context.Context, cycleState *framework.CycleState, pod *v1.Pod) (*framework.PreFilterResult, *framework.Status) {
 	s := getContainerPorts(pod)
+	if len(s) != 0 {
+		return nil, framework.NewStatus(framework.Skip)
```

Comment:
Plz add the unit test to confirm it.

Reply:
Of course. It is because the trigger for the review request was activated once the PR was raised. I am still working on it so I will write up the details of the PR again and mentions when I want a review.

Reply:
To label work-in-progress.
/retitle [WIP] feat: add skip status to nodeports.go